v2/internal/testcommon/creds.go

```diff
@@ -18,17 +18,20 @@ import (
 // this is shared between tests because
 // instantiating it requires HTTP calls
 var cachedCreds azcore.TokenCredential
-var cachedSubID AzureIDs
+var cachedIds AzureIDs
+
+const TestBillingIDVar = "TEST_BILLING_ID"
 
 type AzureIDs struct {
-	subscriptionID string
-	tenantID       string
+	subscriptionID   string
+	tenantID         string
+	billingInvoiceID string
```

Comment:
Same, These changes and some below also exist in your other PR

Reply:
Yeah I need to merge the other one and rebase this, will do.

Reply:
Fixed this and pushed